website/docs/r/replication_controller.html.markdown

```diff
@@ -0,0 +1,574 @@
+---
+layout: "kubernetes"
+page_title: "Kubernetes: kubernetes_replication_controller"
+sidebar_current: "docs-kubernetes-resource-replication-controller"
+description: |-
+  A Replication Controller ensures that a specified number of pod “replicas” are running at any one time. In other words, a Replication Controller makes sure that a pod or homogeneous set of pods are always up and available. If there are too many pods, it will kill some. If there are too few, the Replication Controller will start more.
+---
+
+# kubernetes_replication_controller
+
+A Replication Controller ensures that a specified number of pod “replicas” are running at any one time. In other words, a Replication Controller makes sure that a pod or homogeneous set of pods are always up and available. If there are too many pods, it will kill some. If there are too few, the Replication Controller will start more.
+
+
+## Example Usage
+
+```hcl
+resource "kubernetes_replication_controller" "example" {
+  metadata {
+    name = "terraform-example"
+    labels {
+      Test = "MyExampleApp"
+    }
+  }
+
+  spec {
+    selector {
+      Test = "MyExampleApp"
+    }
+    template {
+      container {
+        image = "nginx:1.7.8"
+        name  = "example"
+
+        resources{
+          limits{
+            cpu    = "0.5"
+            memory = "512Mi"
+          }
+          requests{
+            cpu    = "250m"
+            memory = "50Mi"
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `metadata` - (Required) Standard replication controller's metadata. More info: https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#metadata
+* `spec` - (Required) Spec defines the specification of the desired behavior of the replication controller. More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#spec-and-status
```

Comment:
Good catch, this is kind of a side effect of K8S having these outdated links in swagger/API specs from which I've generated the schema and then these docs. 😢 

I'll fix it here manually and then `sed` the existing docs too in a separate PR.